packages/db/src/migrations/0019_create_offline.ts

```diff
@@ -0,0 +1,15 @@
+import { Kysely, sql } from "kysely"
+import { createTableWithDefaults } from "../utils"
+
+export async function up(db: Kysely<any>) {
+  await createTableWithDefaults("offline", { id: true, createdAt: true }, db.schema)
+    .addColumn("title", sql`character varying(1000)`, (col) => col.notNull())
+    .addColumn("published", sql`timestamp with time zone`, (col) => col.notNull())
+    .addColumn("file", sql`character varying(255)`, (col) => col.notNull())
+    .addColumn("image", sql`character varying(255)`, (col) => col.notNull())
+    .execute()
```

Comment:
Maybe better to let url columsn just be `text`?

Reply:
Agreed